### DISCUSSION TITLE
Is it normal to be queried for password before every snapshot is taken?

### DISCUSSION THREAD

**Discussion by hp3-x** [psy0rz/zfs_autobackup#283] [Q&A]
I'm running the zfs-autobackup command from a root shell: But get bumped for passwd for every snapshot
zfs-autobackup -v --ssh-source lw  lw5x
  zfs-autobackup v3.3-rc.1 - (c)2022 E.H.Eefting (edwin@datux.nl)
  
    Current time               : 2024-12-25 15:41:50
  Selecting dataset property : autobackup:lw5x
  Snapshot format            : lw5x-%Y%m%d%H%M%S
  Timezone                   : Local
  Hold name                  : zfs_autobackup:lw5x
  
  #### Source settings
  [Source] SSH to: lw
  [Source] Keep the last 10 snapshots.
  [Source] Keep every 1 day, delete after 1 week.
  [Source] Keep every 1 week, delete after 1 month.
  [Source] Keep every 1 month, delete after 1 year.
  
  #### Selecting
root@lw's password: 
  [Source] p0: Selected
  [Source] p0/Images-wrk: Selected
  [Source] p0/git: Selected
  [Source] p0/local: Selected
  [Source] p0/pub: Selected
  [Source] p0/srv: Selected
  
  #### Snapshotting
root@lw's password: 
root@lw's password: 
root@lw's password: 
root@lw's password: 
root@lw's password: 
root@lw's password: 
  [Source] Creating snapshots lw5x-20241225154201 in pool p0
root@lw's password: 
  
  #### Thinning source
  
  #### All operations completed successfully
  (No target_path specified, only operated as snapshot tool.)

**Comment by psy0rz** [comment]
Read the getting started manual about setting up ssh keys.
